### PR TITLE
Update marketing copy on GW product page

### PR DESCRIPTION
**Pull request by lucymonie**
## Why are you doing this?
This adds missing marketing copy updates in the following [**Trello Card**](https://trello.com/c/mz1Uw6d4/2511-update-copy-on-gw-product-page)

## Changes
* Page hero text
* Page headline
* First paragraph title
* First paragraph copy

### Old copy
Screen Shot 2019-08-15 at 17 32 17

### New copy
Screen Shot 2019-08-15 at 17 32 05

### CODE REVIEW

File: support-frontend/assets/pages/weekly-subscription-landing/components/hero/hero.jsx [hunk 1]
```diff
@@ -78,7 +78,7 @@ const CampaignHeader = () => (
   <ProductPagehero
     appearance="campaign"
     overheading="Guardian Weekly subscriptions"
-    heading="The Guardian's essential news magazine"
+    heading="Pause for thought with the Guardian's essential news magazine"
```
Comment (jfsoul) [inline support-frontend/assets/pages/weekly-subscription-landing/components/hero/hero.jsx]: I think it is `The Guardian` with a capital T - although I have lost track of the number of times I've seen this debated 😆

File: support-frontend/assets/pages/weekly-subscription-landing/weeklySubscriptionLanding.jsx [hunk 2]
```diff
@@ -90,11 +90,11 @@ const content = (
...
-        <Text title="Open up your world view, Weekly">
-          <LargeParagraph>Inside the essential magazine from
-          The&nbsp;Guardian, you&rsquo;ll find expert opinion, insight and culture, curated to
-          bring you a progressive, international perspective. You&rsquo;ll also discover
-          challenging new puzzles every week. Subscribe today and get delivery worldwide.
+        <Text title="Catch up on the issues that matter">
+          <LargeParagraph>The Guardian Weekly magazine is a round-up of the world news,&#32;
+            opinion and long reads that have shaped the week. Inside, the past seven days&apos;&#32;
+            most memorable stories are reframed with striking photography and insightful companion&#32;
+            pieces, all handpicked from the Guardian and the Observer.
```
Comment (jfsoul) [inline support-frontend/assets/pages/weekly-subscription-landing/weeklySubscriptionLanding.jsx]: Capitals on `The Guardian and The Observer` too

**Comment by jfsoul** [comment]
I noticed that the page Hero has been changed to `The Guardian Weekly` in that document from the trello card.  Do you think that's intentional?

File: support-frontend/assets/pages/weekly-subscription-landing/components/hero/hero.jsx (hunk 1 above)
Comment (johnduffell) [reply support-frontend/assets/pages/weekly-subscription-landing/components/hero/hero.jsx]: blame garnett design updates for that!

**Comment by johnduffell** [comment]
is that the right trello card, I feel like that one that's linked is the one for the other PR?

**Comment by lucymonie** [comment]
Thank god you guys are checking this! 🙏 

@jfsoul, do you mean that this text should be 'The Guardian Weekly'?
Screen Shot 2019-08-15 at 16 45 29

File: support-frontend/assets/pages/weekly-subscription-landing/weeklySubscriptionLanding.jsx (hunk 2 above)
Comment (lucymonie) [reply support-frontend/assets/pages/weekly-subscription-landing/weeklySubscriptionLanding.jsx]: Really? That's so awkward 😬 

Not doubting you, but my inner editor is horrified.

**Comment by lucymonie** [comment]
I checked the Guardian style guide and it is lower case for 'the'

https://www.theguardian.com/guardian-observer-style-guide-t

Screen Shot 2019-08-15 at 16 51 18

**Comment by jfsoul** [comment]
@lucymonie that is what I meant (re: hero), but I am not sure I trust it.

Could we double check with Joel about lowercasing (pointing out it's in the style guide) and the page hero?

**Comment by lucymonie** [comment]
Yes, @jfsoul, shall do. I have made the hero change anyhow, but will double check.

**Comment by lucymonie** [comment]
Joel says yes to the hero text and no to the lowercase 'the'. Apparently the style guide is out of date since the rebrand, and the sign-off is sufficiently high level that we should proceed with the text as it is in the document.

**Pull request merged**